src/test/fakes.rs

```diff
@@ -251,8 +251,16 @@ impl<'a> FakeRelease<'a> {
         // In real life, these would be highlighted HTML, but for testing we just use the files themselves.
         for (source_path, data) in &self.source_files {
             if let Some(src) = source_path.strip_prefix("src/") {
-                let updated = ["src", &package.name, src].join("/");
-                rustdoc_files.push((Box::leak(Box::new(updated)), data));
+                let mut updated = ["src", &package.name, src].join("/");
+                updated += ".html";
+                let source_html = format!(
+                    "<html><head></head><body>{}</body></html>",
+                    std::str::from_utf8(data).expect("invalid utf8")
+                );
+                rustdoc_files.push((
+                    Box::leak(Box::new(updated)),
+                    Box::leak(source_html.into_bytes().into_boxed_slice()),
+                ));
```

Comment:
This took me *so* long to debug - the HTML rewriter doesn't actually do anything unless at least these tags exist.